.github/workflows/assign-PR-author.yml

```diff
@@ -0,0 +1,22 @@
+name: Assign PR author
```

Comment:
I would recommend renaming this file to `PR.yml` and naming the workflow "PR" as well. In the future, we can add any other PR-related automation in this file too.

Reply:
I have renamed both.